demo/main.cpp

```diff
@@ -447,8 +447,8 @@ void RtpBinding::net_ready(int offset)
 
     while(socketGroup->socket[offset].hasPendingDatagrams())
     {
-        int size = (int)socketGroup->socket[offset].pendingDatagramSize();
-        QByteArray rawValue(size, offset);
+        int size = int(socketGroup->socket[offset].pendingDatagramSize());
+        QByteArray rawValue(size, char(offset));
```

Comment:
doesn't make sense for me. not before, not now.
it has to be some somthing like 
```
QByteArray rawValue;
rawValue.resize(size);
```
it will also avoid array initialization which is not needed in this case

Reply:
done